samples/05.custom-components/k.per-message-avatar/README.md

```diff
@@ -0,0 +1,371 @@
+# Sample - Customize per-message avatar
```

Comment:
Need to update and/or delete the original avatar img/initials samples

Reply:
Original sample is unrelated to the new change.

If devs are not customizing the avatar, they should still refer to this sample. If devs customizing avatar, they still need to refer to the original sample if they are adding the overlay.

Adding original sample to Further Reading section.